src/SFML/Window/Unix/KeyboardImpl.cpp

```diff
@@ -46,10 +48,10 @@
 namespace
 {
 
-const KeyCode          nullKeyCode = 0;
-const int              maxKeyCode  = 256;
-KeyCode                scancodeToKeycode[static_cast<std::size_t>(sf::Keyboard::Scan::ScancodeCount)]; ///< Mapping of SFML scancode to X11 KeyCode
-sf::Keyboard::Scancode keycodeToScancode[maxKeyCode]; ///< Mapping of X11 KeyCode to SFML scancode
+const KeyCode                                                                     nullKeyCode = 0;
+const int                                                                         maxKeyCode  = 256;
+sf::priv::EnumArray<sf::Keyboard::Scancode, KeyCode, sf::Keyboard::ScancodeCount> scancodeToKeycode; ///< Mapping of SFML scancode to X11 KeyCode
+std::array<sf::Keyboard::Scancode, maxKeyCode> keycodeToScancode; ///< Mapping of X11 KeyCode to SFML scancode
```

Comment:
Why isn't this an `EnumArray` as well?

Reply:
`KeyCode` type from Xlib is a typedef to `unsigned char`, but `EnumArray` is meant to be used with enumerations.